contracts/test/GrantRound.test.ts

```diff
@@ -0,0 +1,153 @@
+// --- External imports ---
+import { artifacts, ethers, waffle } from 'hardhat';
+import { Artifact } from 'hardhat/types';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
+import { expect } from 'chai';
+
+// --- Our imports ---
+import { GrantRegistry } from '../typechain/GrantRegistry';
+import { timeTravel } from './utils';
+import IERC20Artifact from '../artifacts/@openzeppelin/contracts/token/ERC20/IERC20.sol/IERC20.json';
+
+// --- Parse and define helpers ---
+const { deployContract, deployMockContract } = waffle;
+const { isAddress } = ethers.utils;
+
+describe('GrantRound', function () {
+  let deployer: SignerWithAddress,
+    grantRoundOwner: SignerWithAddress,
+    donor: SignerWithAddress,
+    grantPayee: SignerWithAddress,
+    mpUser: SignerWithAddress;
+  let registry: GrantRegistry;
+  let roundContract: any;
+  let mockERC20: any;
+  const balances: string[] = ['100', '200'];
+  const startTime: number = Math.floor(new Date().getTime() / 1000); // time in seconds
+  const endTime: number = startTime + 86400; // One day
+
+  beforeEach(async () => {
+    const mockUrl: string = 'https://test.url';
+    const minContribution: number = 50;
+    [deployer, grantRoundOwner, donor, grantPayee, mpUser] = await ethers.getSigners();
+    mockERC20 = await deployMockContract(deployer, IERC20Artifact.abi);
+    // Seed funds for matching pool user
+    await mockERC20.mock.transfer.withArgs(mpUser.address, ethers.utils.parseEther(balances[0]));
+    // Seed funds for initial donor
+    await mockERC20.mock.transfer.withArgs(donor.address, ethers.utils.parseEther(balances[0]));
```

Comment:
Are these mock calls doing anything? If I comment them out tests still pass, and based on [the docs](https://ethereum-waffle.readthedocs.io/en/latest/mock-contract.html) it seems these need one more method chained on